x-pack/plugins/reporting/server/config/index.ts

```diff
@@ -18,10 +18,10 @@ export const config: PluginConfigDescriptor<ReportingConfigType> = {
   exposeToBrowser: { poll: true, roles: true },
   schema: ConfigSchema,
   deprecations: ({ unused }) => [
-    unused('capture.browser.chromium.maxScreenshotDimension'), // unused since 7.8
-    unused('poll.jobCompletionNotifier.intervalErrorMultiplier'), // unused since 7.10
-    unused('poll.jobsRefresh.intervalErrorMultiplier'), // unused since 7.10
-    unused('capture.viewport'), // deprecated as unused since 7.16
+    unused('capture.browser.chromium.maxScreenshotDimension', { level: 'critical' }), // unused since 7.8
+    unused('poll.jobCompletionNotifier.intervalErrorMultiplier', { level: 'critical' }), // unused since 7.10
+    unused('poll.jobsRefresh.intervalErrorMultiplier', { level: 'critical' }), // unused since 7.10
+    unused('capture.viewport', { level: 'critical' }), // deprecated as unused since 7.16
```

Comment:
I recommend using the `warning` level for the unused config settings. A setting in kibana.yml that doesn't do anything is not going to break anything.